pyproject.toml

```diff
@@ -1,3 +1,91 @@
+[tool.poetry]
+name = "cookiecutter"
+version = "2.0.2"
+
+description = """\
+  A command-line utility that creates projects from project \
+  templates, e.g. creating a Python package project from a \
+  Python package project template.\
+  """
+
+authors = [
+    "Audrey Roy Greenfeld <audreyr@gmail.com>"
+]
+
+license = "BSD"
+readme = "README.md"
+
+packages = [
+    { include = "cookiecutter" }
+]
+
+homepage = "https://github.com/cookiecutter/cookiecutter"
+documentation = "https://cookiecutter.readthedocs.io"
+
+keywords = [
+    "cookiecutter",
+    "Python",
+    "projects",
+    "project templates",
+    "Jinja2",
+    "skeleton",
+    "scaffolding",
+    "project directory",
+    "package",
+    "packaging"
+    ]
+
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Environment :: Console",
+    "Intended Audience :: Developers",
+    "Natural Language :: English",
+    "License :: OSI Approved :: BSD License",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Programming Language :: Python :: Implementation :: PyPy",
+    "Programming Language :: Python",
+    "Topic :: Software Development",
+]
+
+
+[tool.poetry.build]
+generate-setup-file = false
+
+
+[tool.poetry.dependencies]
+python = ">=3.6"
+
+binaryornot = ">=0.4.4"
+Jinja2 = ">=2.7,<4.0.0"
+click = ">=7.0,<8.0.0"
+pyyaml = ">=5.3.1"
+jinja2-time = ">=0.2.0"
+python-slugify = ">=4.0.0"
+requests = ">=2.23.0"
+
+
+[tool.poetry.dev-dependencies]
+pytest = "*"
+pytest-cov = "*"
+pytest-mock = "*"
+freezegun = "*"
+
+
+[tool.poetry.scripts]
+cookiecutter = "cookiecutter.__main__:main"
+
+
+[build-system]
```

Comment:
This is breaking the `tool` table into what is called an "out-of-order" tables. The [TOML spec discourages it](https://toml.io/en/v1.0.0#table):

> Defining tables out-of-order is discouraged.
> 
> ```toml
> # VALID BUT DISCOURAGED
> [fruit.apple]
> [animal]
> [fruit.orange]
> ```
>
> ```toml
> # RECOMMENDED
> [fruit.apple]
> [fruit.orange]
> [animal]
> ```

Personally, I've hit some nastly bugs with toiml parsers in the past with out-of-order tables. I would strongly recommend moving this table outside of the `tool` table.

Reply:
thank you